### ISSUE TITLE
compile error after upgrade to xCode Version 9.3 (9E145)

### ISSUE THREAD

**Issue by lincanli** [fortmarek/SwipeViewController#30]
On InterfaceController.swift -->

``` swift
if buttonDelegate.equalSpaces {
            //Space between buttons
            buttonDelegate.x = (delegate.viewWidth - 2 * buttonDelegate.offset - totalButtonWidth) / CGFloat(buttons.count + 1)
        }
```

error goes:

```
Overlapping accesses to 'buttonDelegate', but modification requires exclusive access; consider copying to a local variable
```

**Comment by relaxsus** [comment]
How can I solve this problem? after the update the same error

**Comment by fortmarek** [comment]
Fixed with version 1.1.1, error occurred because of new Swift 4.1 changes.

**Comment by shshemi** [comment]
Is version 1.1.1 available to install using cocoa pods?
